apis/Google.Cloud.Spanner.V1/Google.Cloud.Spanner.V1/TransactionPool.cs

```diff
@@ -115,6 +115,16 @@ public static TransactionOptions GetLastUsedTransactionOptions(this Session sess
             return null;
         }
 
+        public static bool IsPreWarmedTransactionReady(this Session session)
+        {
+            SessionInfo info;
+            if (s_sessionInfoTable.TryGetValue(session, out info))
+            {
+                return info.HasActiveTransaction;
+            }
+            return false;
+        }
+
```

Comment:
overdoing expression bodies has so far caused more work than its worth.  for cases where its probable to add additional logic, I prefer to leave it as statement syntax